### DISCUSSION TITLE
UI opposite day

### DISCUSSION THREAD

**Discussion by chriso86** [gitextensions/gitextensions#9549] [General]
Hey guys,

I'm not sure if it's just me.
I just updated, and it seems that the local branch and remote branch colors are switched, and the Ok and Abort buttons are switched.... are we being trolled?

Lol. Weird UX decisions.

**Comment by RussKie** [comment]
Yes, this was intended, changed to use the same color scheme as git-log command line.
This was much debate about this change both before and after the release and you are not the first to ask...

**Reply by chriso86** [reply]
I know exactly what you're talking about.
It's an adjustment for existing users, but good for new users.
I get it, and you did make the correct decision as far as I'm concerned.
But it's weird to make the change so abruptly, UX changes sometimes take time to reapply new programming to the minds of the users.
